### ISSUE TITLE
Threshold B

### ISSUE THREAD

**Issue by ja-thomas** [PhilippPro/OpenML-R-Bot#1]
I don't understand what this means... like at all...
Can you maybe rephrase @PhilippPro @DanielKuehn87 

....
They are chosen by the following procedure: for each algorithm, a threshold $B$ is set (see below) and, if the number of results for a dataset exceeds $B$, we draw randomly $B$ of the results obtained for this algorithm and this dataset. For each algorithm, the threshold value $B$ is chosen for each algorithm separately to exactly obtain 500000 results for each algorithm.

**Comment by PhilippPro** [comment]
This is difficult to phrase. Is it really not understandable? 

> We take a maximum of $B_i$ results for each dataset and algorithm $i$ combination. This threshold $B_i$ is set for each algorithm $i$ so that we exactly obtain 500000 results for each algorithm. 

Maybe these two sentences are better understandable?

**Comment by ja-thomas** [comment]
(hier können wir ja auch deutsch schreiben :laughing: )

Es ist irgendwie etwas komisch dass gesagt wird:
 ".. threshold B (see below) ..." was dann aber gar nicht mehr vorkommt.

Wenn ich es richtig verstehe geht ihr so vor oder:

Jeder Algorithmus soll gleich oft vorkommen und zwar so dass über alle Algos 500K Ergebnisse da sind. Wenn es für einen oder mehrere Algos zu wenig Ergebnisse gibt (e.g. knn) wird die Anzahl der Ergebnisse der anderen Algos gleichmäßig erhöht bis insgesamt 500K erreicht sind. 
 

Korrekt?

**Comment by PhilippPro** [comment]
Nein. Sagen wir wir haben einen Algorithmus, z.B. SVM. Wir nehmen zunächst alle Ergebnisse die es gibt von SVM. Dann streichen wir von den Datensätzen mit sehr vielen Ergebnissen so viele Ergebnisse, bis für SVM insgesamt genau 500000 Ergebnisse da sind. Der Cutoff wird so gesetzt, dass genau 500000 Ergebnisse rauskommen.

**Comment by ja-thomas** [comment]
ist das nicht das gleiche? (Nur andersrum formuliert)

**Comment by PhilippPro** [comment]
Bei dir klang es so, also ob du über alle Algos hinweg 500000 Ergebnisse haben möchtest. Es soll aber für jeden Algorithmus 500000 Ergebnisse geben (bis auf kknn).
